x/ccv/provider/keeper/keeper.go

```diff
@@ -621,6 +621,70 @@ func (k Keeper) DeleteInitChainHeight(ctx sdk.Context, chainID string) {
 	store.Delete(types.InitChainHeightKey(chainID))
 }
 
+// SetPendingVSCs sets in store the list of pending ValidatorSetChange packets under chain ID
+func (k Keeper) SetPendingVSCs(ctx sdk.Context, chainID string, packets []ccv.ValidatorSetChangePacketData) {
```

Comment:
Clarification: For CCV v1, why is there a list here when all mature child chains get the same updates? Is it because chains that have just finished initializing may have a different update?

Reply:
> Is it because chains that have just finished initializing may have a different update?

Yes. The CCV channels to different consumer chains get established async. This means that for each consumer chain there will be a list of VSC packets with validator updates that occurred during the initialization of that consumer chain.